src/reducers/craReducer.js

```diff
@@ -65,6 +96,32 @@ export default function cra(state = initialState, action) {
           ...state.errorsRequired,
           canal: false },
       };
+    case 'DELETE_CANAL_VALUE':
+      // eslint-disable-next-line no-undef-init
+      let canal = undefined;
```

Comment:
Normalement pas besoin : let canal; => vaudra direct undefined
```suggestion
      let canal;
```